reconstruct/reconstruct_test.go

```diff
@@ -23,7 +23,7 @@ func Test_reconstruct(t *testing.T) {
 			log.Fatal("Couldn't read file")
 		}
 		fasta.Write("RandGeneOutput.fasta", simulate.RandGene("test", test.length, GCcontent)) //galGal6 GC
-		simulate.Simulate("RandGeneOutput.fasta", tre)
+		simulate.Simulate("RandGeneOutput.fasta", tre, "newTestFiles/genePred.gp")
```

Comment:
There is a convention to put test files in a folder called "testdata", which the go tools know to ignore when looking for code.  Could you change this from newTestFiles to testdata?